src/argilla_llama_index/llama_index_handler.py

```diff
@@ -89,177 +94,127 @@ def __init__(
 
             self.ARGILLA_VERSION = rg.__version__
 
-        except ImportError:
+        except ImportError as e:
```

Comment:
i think we can skip this because the package is an installable and comes with argilla included, right?